react/features/base/settings/middleware.js

```diff
@@ -52,15 +53,15 @@ function _maybeSetAudioOnly(
 function _updateLocalParticipant(store) {
     const state = toState(store);
     const localParticipant = getLocalParticipant(state);
-    const profile = state['features/base/profile'];
+    const settings = getSettings(state);
```

Comment:
Adjust the jsdoc in this function, it still mentions a "profile".

Reply:
Sh*t I thought I found all occasions :( thanks!